### PR TITLE
Added release notes for v5.4.1

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: docs/modules/ROOT/pages/release-notes-5.4.1.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+= 5.4.1 Release Notes
+
+== New Features
+
+There are no new features in this release.
+
+== Fixes
+
+* Custom template actions aren't loaded when a template is cloned initially.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.4.1.adoc:9]: Does this mean:

> Custom template actions are not initially loaded when a template is cloned

Or:

> Custom template actions aren't loaded when a template is initially loaded

?

**Review by rebekah-lawrence — APPROVED** [review]
I think a slight clarification is needed; I wasn't sure which way to read it. Otherwise, LGTM

Approving in advance

File: docs/modules/ROOT/pages/release-notes-5.4.1.adoc (hunk 1 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.4.1.adoc:9]: It's "Custom template actions are not initially loaded when a template is cloned"

Comment (rebekah-lawrence) [reply docs/modules/ROOT/pages/release-notes-5.4.1.adoc:9]:
```suggestion
* Custom template actions aren't initially loaded when a template is cloned.
```

**Pull request merged**